app/assets/javascripts/routes/products_handlers.js

```diff
@@ -99,6 +100,11 @@ let ProductsHandlers = {
       type: ActionTypes.PRODUCT_RECEIVE,
       product: product
     });
+
+    Dispatcher.dispatch({
+      type: ActionTypes.BOUNTY_SELECTED_CHANGED,
```

Comment:
Expected ',' and instead saw 'type'.<br>Expected an identifier and instead saw ':'.<br>Expected ',' and instead saw 'ActionTypes'.<br>Expected an identifier and instead saw '.'.<br>Expected ',' and instead saw 'BOUNTY_SELECTED_CHANGED'.<br>Expected an identifier and instead saw ','.